Automattic/wp-calypso
# Components: Move search into @automattic/components

#### Changes proposed in this Pull Request

* Moves the `search` component into `@automattic/components` package now that the WordPress monorepo update is completed and the SSR issues are resolved.

After this is merged I will open another PR to move the language picker into `@automattic/components`

#### Testing instructions



* Test several entrypoints to ensure that SSR is working
* Test the language picker at `/me/account` and ensure that search continues to work.
* Run storybook and test the search component's story
